User data source (#85)
Axual/terraform-provider-axual: DEVELOPERS.md

    - `GranularBrowsePermission` enabled
    - `ConnectSupport` enabled
- Then in the [`test_config.yaml`](./internal/tests/test_config.yaml), replace the `groupName` to a Group you are a member of.

> - Then in the [`test_config.yaml`](./internal/tests/test_config.yaml), replace the `userEmail` to an email of your logged-in user.

Reviewer: Let's clarify that we want to the `email` of the test-user

```suggestion
- Then in the [`test_config.yaml`](./internal/tests/test_config.yaml), replace the `userEmail` to your email identifying your user.
```

BTW, this works also with an email identifying any other user than the User configured in the providers.tf
